cmd/soroban-cli/tests/it/config.rs

```diff
@@ -0,0 +1,156 @@
+use crate::util::{temp_dir, Sandbox, SorobanCommand};
+use std::{fs, path::Path};
+
+#[test]
+fn set_and_remove_network() {
+    let dir = temp_dir();
+
+    let _ = Sandbox::new_cmd("config")
+        .current_dir(&dir)
+        .arg("network")
+        .arg("add")
+        .arg("--rpc-url")
+        .arg("https://127.0.0.1")
+        .arg("--network-passphrase")
+        .arg("Local Sandbox Stellar Network ; September 2022")
+        .arg("local")
```

Comment:
Shouldn't the network name be first? `network add local ...` or I guess it doesn't matter?

Reply:
Yeah it's a positional arg so the flag args can be before or after.  I'll swap to make this clearer.

Reply:
All good, just double checking.